### ISSUE TITLE
Edit x and y axis labels

### ISSUE THREAD

**Issue by meghdadFar** [PaddlePaddle/VisualDL#563]
Is there a way to change the name of the x and y axis to a custom name?
Also, can I control the plot colors?

**Comment by linshuliang** [comment]
The name of x axis is fixed to be **Step**,  however, you can set the name of y aixs

Take an example:

```
# coding=utf-8
from visualdl import LogWriter
log_writer = LogWriter("log", sync_cycle=20)

# create an instance of scalar and set the name of y axis to be "acc"/ "loss" with mode "train"
with log_writer.mode("train") as logger:
    train_acc = logger.scalar("acc")
    train_loss = logger.scalar("loss")

# create an instance of scalar and set the name of y axis to be "acc" with mode "test"
with log_writer.mode("test") as logger:
    test_acc = logger.scalar("acc")

# log data
for i in range(1000):
    train_acc.add_record(step, i / 1000.0)
    train_loss.add_record(step, 1000.0 / (1000 + i))
    test_acc.add_record(step, 1 -  i / 1000.0)
```  

The display on the web:

https://github.com/PaddlePaddle/VisualDL/blob/develop/demo/component/usage-interface/scalar-interface.png

There is no options to control the colors too.
